lib/yotpo/version.rb

```diff
@@ -1,3 +1,3 @@
 module Yotpo
-  VERSION = '1.0.1'
+  VERSION = '1.0.2'
```

Comment:
I would put the version change in a separate commit / PR.

Reply:
Why did Yotpo not use this PR? Coming to this gem now, and it seems crazy to provide a known key on each and every call.